peak/mapper/mapper.py

```diff
@@ -79,22 +80,23 @@ def gen_mapping(
         return
     def f_fun(inst):
         return all(constraint(inst) for constraint in constraints)
+    bv_isa_list = list(filter(f_fun, bv_isa.enumerate()))
```

Comment:
Maybe just a comment about why we're keeping two lists, and that its a hack that should go away once we figure out how to do the conversions.

Reply:
Added comment ont he bv_isa argument